db/migrate/20220513153429_rename_about_pages_to_topical_event_about_pages.rb

```diff
@@ -0,0 +1,5 @@
+class RenameAboutPagesToTopicalEventAboutPages < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
That's a really good point.  I believe Rails won't run this migration if there's a more recent migration that has been run.

I'll rename the migration to have a date that is more recent, just in case.

Reply:
I think that's my understanding too, although would tests be passing if that was the case? 🤔 not sure
In any case, makes sense to update the migration anyway as you've just done 😃

Reply:
I'm pretty sure that in CI, the database gets rebuilt from the `db/schemas.rb` file each time (rather than running the migrations).

Locally, I ran the migration before rebasing against the main branch, so the migration got run anyway.

Reply:
ahhhh that explains it, thanks!